extra/sigmund~/sigmund~.c

```diff
@@ -488,9 +488,9 @@ static void sigmund_getpitch(int npeak, t_peak *peakv, t_float *freqp,
 
         /* first guess by parabolic peak fitting */
     fbestbin = bestbin + (ppt[bestbin+1].p_weight
-        - ppt[bestbin-1].p_weight) /
+        - ppt[bestbin == 0 ? 0 : bestbin-1].p_weight) /
```

Comment:
shouldn't that be `bestbin < 1` (to catch all kinds of invalid values)?

Reply:
and do the clamping only once (so we don't have to repeat the code in https://github.com/pure-data/pure-data/pull/2344/files#diff-c5ff8229d6a761a660b6ac33ae780bd8c92f30d9e4fa2616a0653d88e845458bR493)

Reply:
So perhaps it makes sense simply to toss any peak at bin 0... or perhaps it's worth writing some sort of special case code for it?